src/ui/RelevanceInspector/AgGrid.ts

```diff
@@ -0,0 +1,62 @@
+import { $$ } from '../../utils/Dom';
```

Comment:
The library is nearly 900kb. If I included it, it would increase the file size in eager mode by a very significant amount, for a debugging component.

I prefer to manually import the script on demand, and there's no easy way that I know of to do that using webpack functionalities directly to include external resources from a CDN.

Reply:
Ok, makes sense. The downside I see is that we will not be able to leverage tree shaking using this method, but it's a trade off that seems ok.